src/AbstractFFTs.jl

```diff
@@ -5,7 +5,7 @@ import ChainRulesCore
 export fft, ifft, bfft, fft!, ifft!, bfft!,
        plan_fft, plan_ifft, plan_bfft, plan_fft!, plan_ifft!, plan_bfft!,
        rfft, irfft, brfft, plan_rfft, plan_irfft, plan_brfft,
-       fftshift, ifftshift, fftshift!, ifftshift!, Frequencies, fftfreq, rfftfreq
+       region, fftshift, ifftshift, fftshift!, ifftshift!, Frequencies, fftfreq, rfftfreq
```

Comment:
```suggestion
       fftshift, ifftshift, fftshift!, ifftshift!, Frequencies, fftfreq, rfftfreq
```
let's be conservative and not export it for now; it is still part of the API since it's documented